src/store/personalia/reducer.ts

```diff
@@ -61,15 +101,25 @@ const personaliaReducer: Reducer<PersonaliaState, PersonaliaActions> = (state =
 
 function* hentPersonalia(action: HentPersonaliaAction) {
     try {
-        const response = yield call( () => fetchPersonaliaData(action.fnr));
+        const response = yield call(() => fetchPersonaliaData(action.fnr));
         yield put(hentPersonaliaSuccess(response));
     } catch (e) {
         yield put(hentPersonaliaError(e));
     }
 }
 
+function* hentInnloggetNivaa4(action: HentHarbruktNivaa4Action) {
```

Comment:
why to use generator function?

Reply:
I believe generator function is being used to make code work asynchronically. Basically this generator function is initiating the action and storing the response, so it is performing two actions and second has dependency on the first one. yeild has special relation with generator function which makes the function pause. That means other function can not interrupt in between when the expression inside yeild is in process, which makes code efficient. This is my understanding, I dont have much experience with redux-saga.

Reply:
@dragutin-nav Its not really my choice, generator function is required to use redux-saga which is consistently used in this repository. Im not so familiar with it either but did not want to introduce a second way of doing things

Reply:
Thanks @sneha-d-desai , @izisfro. I need to learn about generator functions. The rest of the code seems good.